mantidproject/mantid
# Allow overwriting members of a WorkspaceGroup when setting output properties to the same workspace name

Fixes #12717 

This adds the ability for an algorithm to overwrite the members of a WorkspaceGroup when processing workspace groups when an output property is given the same name as an input property, before this would have created new workspaces with `name_n` style names, which is inconsistent with what you would expect given how algorithms run over single workspaces.

To test:
- Code review
- Try some algorithms that take multiple input and output workspaces and see that it behaves as expected
